### DISCUSSION TITLE
AttributeError: `dense` is not available with Keras 3

### DISCUSSION THREAD

**Discussion by simwes** [lululxvi/deepxde#1730] [General]
Hello,

I have just install DeepXDE following the official [instruction](https://deepxde.readthedocs.io/en/latest/user/installation.html). However, when I run any tutorial script I get the following error:
`AttributeError: `dense` is not available with Keras 3.
`  

Can anyone help with that?

Thank you!

**Comment by vl-dud** [comment]
Hello! Try to downgrade keras:
```
pip uninstall keras
pip install keras==2.15.0
```

**Reply by simwes** [reply]
Hi, 

Thank you for the quick suggestion. I also had to downgrade tensorflow to 2.15.0 version to make it work.
Thank you for your help !!

**Reply by GordonLuo21** [reply]
Thank you for your help!!! I also have solved my problem!

**Reply by 238Ver** [reply]
It works, thanks a lot !

**Comment by lululxvi** [comment]
See https://deepxde.readthedocs.io/en/latest/user/installation.html#installation
